src/Core/Models/View/SendView.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using Bit.Core.Enums;
+using Bit.Core.Models.Domain;
+using Bit.Core.Utilities;
+
+namespace Bit.Core.Models.View
+{
+    public class SendView : View
+    {
+        public SendView(Send send)
+        {
+            Id = send.Id;
+            AccessId = send.AccessId;
+            Type = send.Type;
+            MaxAccessCount = send.MaxAccessCount;
+            AccessCount = send.AccessCount;
+            RevisionDate = send.RevisionDate;
+            DeletionDate = send.DeletionDate;
+            ExpirationDate = send.ExpirationDate;
+            Disabled = send.Disabled;
+            Password = send.Password;
+        }
+
+        public string Id { get; set; }
+        public string AccessId { get; set; }
+        public string Name { get; set; }
+        public string Notes { get; set; }
+        public byte[] Key { get; set; }
+        public SymmetricCryptoKey CryptoKey { get; set; }
+        public SendType Type { get; set; }
+        public SendTextView Text { get; set; } = new SendTextView();
+        public SendFileView File { get; set; } = new SendFileView();
+        public int? MaxAccessCount { get; set; }
+        public int AccessCount { get; set; }
+        public DateTime RevisionDate { get; set; }
+        public DateTime DeletionDate { get; set; }
+        public DateTime? ExpirationDate { get; set; }
+        public string Password { get; set; }
+        public bool Disabled { get; set; }
+        public string UrlB64Key => Key == null ? null : CoreHelpers.Base64UrlEncode(Key);
+        public bool MaxAccessCountReached => MaxAccessCount.HasValue && AccessCount >= MaxAccessCount.Value;
+        public bool Expired => ExpirationDate.HasValue && ExpirationDate.Value <= DateTime.Now;
```

Comment:
On mobile do we use `Now` vs. `UtcNow` where it's using the device-localized time? Not saying this is wrong, just not sure and wanted to ask.

Reply:
I'm glad you caught this. I wanted to discuss and forgot.

I'm not 100% sure it'll matter since I believe these are transmitted as universal, but If not, I'd imagine we want a Send to expire at the same instant everywhere, not an hour ealier/later depending on where the phone happens to be.

Reply:
@mportune-bw , @kspearrin , can you guys weigh in here, perhaps you'll have a quick answer?

Reply:
it might not matter here since dot net should account for timezone of now vs utc. better make it UtcNow, though, since that is what will be coming from the server with this data.